src/common.c

```diff
@@ -443,44 +443,26 @@ get_mentions(gboolean whole_word, gboolean case_sensitive, const char* const mes
 }
 
 gboolean
-call_external(gchar** argv, gchar** std_out, gchar** std_err)
+call_external(gchar** argv)
 {
-    GError* spawn_error = NULL;
-    GError* exit_error = NULL;
+    GError* spawn_error;
     gboolean is_successful;
-    gint wait_status;
-
-    GSpawnFlags flags = G_SPAWN_SEARCH_PATH;
-    if (std_out == NULL)
-        flags |= G_SPAWN_STDOUT_TO_DEV_NULL;
-    if (std_err == NULL)
-        flags |= G_SPAWN_STDERR_TO_DEV_NULL;
-
-    is_successful = g_spawn_sync(NULL, // Inherit the parent PWD.
-                                 argv,
-                                 NULL, // Inherit the parent environment.
-                                 flags,
-                                 NULL, NULL, // No func. before exec() in child.
-                                 std_out, std_err,
-                                 &wait_status, &spawn_error);
 
+    GSpawnFlags flags = G_SPAWN_SEARCH_PATH | G_SPAWN_STDOUT_TO_DEV_NULL | G_SPAWN_STDERR_TO_DEV_NULL;
+
+    is_successful = g_spawn_async_with_pipes(NULL, // Inherit the parent PWD
```

Comment:
Why `g_spawn_async_with_pipes()` and not `g_spawn_async()` ?

Reply:
Because before there were all those arguments, but now they're not used anymore, so the slimmer API `g_spawn_async()` could be used ... which under the hood still calls `g_spawn_async_with_pipes()` https://docs.gtk.org/glib/func.spawn_async.html ... :D

Reply:
i'll change it.